app/views/company/vacancies/_vacancy.html.erb

```diff
@@ -1,21 +1 @@
-<section class="vacancy-item ">
-  <div class="row">
-    <div class="col-md-1 col-xs-5">
-      <%= image_tag("ico-company-default.png", class: 'img-responsive img-company') %>
-    </div>
-    <div class="col-md-9 col-xs-6">
-      <div class="vacancy-title">
-        <span class="text-primary show-title-vacancy">
-          <%= vacancy.job_title %>
-        </span>
-      </div>
-      <div class="vacancy-details">
-        <span><i class="fa fa-briefcase" aria-hidden="true"></i><%= vacancy.company_name %></span>
-        <span><i class="fa fa-map-marker" aria-hidden="true"></i><%= vacancy.location %></span>
-      </div>
-    </div>
-    <div class="col-md-2 col-xs-12">
-      <%= link_to t("messages.edit_vacancy"), edit_vacancy_path(vacancy.id), class: "btn btn-primary btn-border-bottom-primary vertical-25 grow" %>
-    </div>
-  </div>
-</section>
+<%= render 'shared/components/vacancy_item', vacancy: vacancy, link: edit_vacancy_path(vacancy.id) %>
```

Comment:
maybe change the partial to `components/vacancies/vacancy_item`

Reply:
thinking bout components, I suggest `components/vacancy_item`, because makes no sense another component called `vacancy_item`.

Reply:
I agree with @andrelugomes